loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/json/LoadFlowResultJsonModule.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2018, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
```

Comment:
Please update the copyright year. Is the copyright iTesla or RTE btw?

Reply:
`Copyright (c) 2020, RTE (https://ww.rte-france.com)`